core-services/ci-secret-generator/_config.yaml

```diff
@@ -191,21 +191,3 @@
     - build02
     service_account:
     - sprint-automation
-- fields:
```

Comment:
I will start to refresh the tokens (from `secret/config-updater-token-version-1`) in vault manually after merge.